pkg/resource/agents-toggle/observability_bundle_configmap.go

```diff
@@ -59,11 +59,25 @@ func GenerateObservabilityBundleConfigMap(ctx context.Context, lc loggedcluster.
 		return v1.ConfigMap{}, errors.Errorf("unsupported logging agent %q", lc.GetLoggingAgent())
 	}
 
-	if observabilityBundleVersion.GE(semver.MustParse("0.10.0")) {
+	switch lc.GetKubeEventsLogger() {
```

Comment:
You're missing something like this https://github.com/giantswarm/logging-operator/blob/af109fb9070d7ebd9eb002ee4055aeda6bf51117/pkg/resource/logging-agents-toggle/observability_bundle_configmap.go#L36 with bundle 1.9.0 that sets the default to grafana agent :)

Reply:
Done for observability-bundle < 1.9.0